src/modules/SpaceCalculation.rb

```diff
@@ -191,29 +197,21 @@ def EvaluateFreeSpace(spare_percentage)
     end
 
     # return default ext3/4 journal size (in B) for target partition size
-    def DefaultExtJournalSize(part)
-      part = deep_copy(part)
-      if Ops.get_symbol(part, "used_fs", :unknown) == :ext2
+    def DefaultExtJournalSize(filesystem)
+      if type?(:ext2, filesystem)
```

Comment:
AFAICS, there is `Storage.ext4?(filesystem)`, but not `Storage.ext2?(filesystem)` or `Storage.reiserfs?(filesystem)` (for example).

On the other hand, the fact that those methods are directly floating in the `Storage` namespace kind of confirm @jreidinger's concerns about object orientation, in my opinion.